### PR TITLE
Getting Started - Outlined different options for app setup

### PR DESCRIPTION
**Pull request by theangchen**
1) Scaffolding an app
2) Scaffolding with 1 Example
3) Get all the examples
4) Help option

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -18,9 +18,19 @@ npm start
 
 Then open [http://localhost:3000/](http://localhost:3000/) to see your app.
 
+**Option 2:** If you would like to scaffold from 1 example, please see [Running an Example](./building-your-application/available-scripts/running-example.md)
+
+**Option 3:** You can also clone the boilerplate repo to get all the examples. Then run `npm i` followed by `npm start` from the example directory you want to run.
```
Comment (jaygiang) [inline docs/getting-started.md:23]: Maybe can can link https://github.com/NMFS-RADFish/boilerplate to `boilerplate repo` text so it's easier for the reader to find

Comment (jaygiang) [inline docs/getting-started.md:23]: The instructions are a bit misleading. To run an example project, you'll actually need to:

1. cd in `examples` folder
2. cd into an example `ex. computed-form-fields, main, etc.`
3. run `npm i`
4. run `npm start`

File: docs/getting-started.md [hunk 2]
```diff
@@ -18,9 +18,19 @@ npm start
...
+
+**To show help:**
+
+```bash
+npx @nmfs-radfish/create-radfish-app my-app --help
+```
+
 # Creating an PWA
```
Comment (jaygiang) [inline docs/getting-started.md:31]: Found a typo! We should change `Creating an PWA` to `Creating a PWA`.

**Review by jaygiang — APPROVED** [review]
@theangchen Looks great!

Just added a couple of suggestions/potential changes. Good to merge after those changes! ✅

**Review by brockinit — APPROVED** [review]
Looks great! Ty

**Pull request merged**